apps/mark-scan/backend/src/store.ts

```diff
@@ -313,4 +319,12 @@ export class Store {
       ballotsPrintedCount
     );
   }
+
+  createHardwareConfig(): void {
+    this.client.run('delete from hardware_config');
+    this.client.run(
+      'insert into hardware_config (is_pat_device_connected) values (?)',
+      0
+    );
+  }
```

Comment:
Accidentally left in here?

Reply:
Sorry, should have done a last pass for this type of cleanup before opening for review.

https://github.com/votingworks/vxsuite/pull/4284/commits/f977e7e976ffbf77a2e882c749ee8352e1fe3ffe